packages/webapp/src/_app/eos/ual/EdenUALProvider.tsx

```diff
@@ -1,8 +1,19 @@
 import { useEffect, useState } from "react";
 
-import { appName, chainConfig } from "config";
+import { appName, availableWallets, chainConfig } from "config";
 import { anchor, scatter, ledger } from "./config";
 
+const authenticators: any[] = [];
+if (availableWallets.indexOf("ANCHOR") >= 0) {
+    authenticators.push(anchor);
+}
+if (availableWallets.indexOf("SCATTER") >= 0) {
+    authenticators.push(scatter);
+}
+if (availableWallets.indexOf("LEDGER") >= 0) {
+    authenticators.push(ledger);
+}
```

Comment:
`availableWallets.includes("ANCHOR")` is more readable here, IMHO.

Reply:
oooo `.includes`! nice!